examples/ssx-test-serverless-dynamodb-api/src/libs/siwe.ts

```diff
@@ -0,0 +1,98 @@
+import { middyfy } from '@libs/lambda';
+import {
+  SSXServer,
+  SSXInfuraProviderNetworks,
+  SSXRPCProviders,
+} from '@spruceid/ssx-serverless';
+import { handleResponse } from './http';
+import { DynamoDB } from 'aws-sdk';
+
+const DYNAMODB_TABLE_NAME = 'SSX';
+const dynamoDb = new DynamoDB.DocumentClient({
+  region: 'localhost',
+  endpoint: 'http://localhost:8000',
+});
+
+const update = async <T>(key: any, value: any, opts?: Record<string, any>): Promise<T> => {
+  const ddbParams = {
+    TableName: DYNAMODB_TABLE_NAME,
+    Item: { address: key, ...value },
+  };
+  return dynamoDb.put(ddbParams).promise()
+    .then(result => result as T);
+};
+
+const retrieve = async <T>(key: any): Promise<T> => {
+  const ddbGetParams = {
+    TableName: DYNAMODB_TABLE_NAME,
+    Key: { address: key }
+  };
+  return dynamoDb.get(ddbGetParams).promise()
+    .then(({ Item }) => Item as T)
+    .catch(e => e?.message || "Unable to retrieve session.");
+};
+
+const create = async <T>(value: any): Promise<T> => {
+  const ddbParams = {
+    TableName: DYNAMODB_TABLE_NAME,
+    Item: value,
+  };
+  return dynamoDb.put(ddbParams).promise()
+    .then(result => result as T);
+};
+
+const _delete = async <T>(key: any): Promise<T> => {
+  const ddbParams = {
+    TableName: DYNAMODB_TABLE_NAME,
+    Key: { address: key },
+  };
+  return dynamoDb.delete(ddbParams).promise() as T;
+};
+
+const ssx = new SSXServer({
+  daoLogin: true,
+  providers: {
+    rpc: {
+      service: SSXRPCProviders.SSXInfuraProvider,
+      apiKey: process.env.INFURA_ID,
+      network: SSXInfuraProviderNetworks.GOERLI,
```

Comment:
Since this is just an example I think it would just add unnecessary complexity to the code since the main goal is to demonstrate the usage of ssx and not to have a production ready template for it.